features/keychain/module/crypto/secp256k1.js

```diff
@@ -23,15 +20,14 @@ export const create = ({ getPrivateHDKey }) => {
   const curve = new EC('secp256k1')
 
   const createInstance = () => ({
-    signBuffer: async ({ seedId, keyId, data, ecOptions, enc = 'der' }) => {
+    signBuffer: async ({ seedId, keyId, data, enc = 'der' }) => {
```

Comment:
should we throw if `ecOptions` are provided? e.g. from here https://github.com/ExodusMovement/assets/blob/ed2b5397549184fa47ab81d7de5e08b3ec067ccb/ethereum/ethereum-lib/src/sign-message.js#L84

Reply:
Can we hold off for now and just do a message instead? This would throw for all assets.

Reply:
@mvayngrib, no, this was ever used with `ecOptions = { canonical: true }`, and we are perfectly fine/compatible with that

Throwing would be a breaking change, this isn't (as pers wasn't released)

Reply:
@feri42 I don't think we need even a message, it's safe to ignore that option

Reply:
cool, thanks for explaining